tests/autofix_files/trio100.py

```diff
@@ -0,0 +1,84 @@
+# type: ignore
```

Comment:
there's not a ton to see here ("trio100_simple_autofix.py" is easier to read), but if you want to review it you should view this and `tests/eval_files/trio100.py` with a difftool of your choice.